datahub/company/test/test_models.py

```diff
@@ -232,3 +235,32 @@ def test_str(self, first_name, last_name, company_factory, expected_output):
             company=company_factory(),
         )
         assert str(contact) == expected_output
+
+
+class TestCompanyExportCountry:
+    """Tests Company export country model"""
+
+    def test_str(self):
+        """Test the human friendly string representation of the object"""
+        export_country = CompanyExportCountryFactory()
+        status = f'{export_country.company} {export_country.country} {export_country.status}'
+        assert str(export_country) == status
+
+    def test_unique_constraint(self):
+        """
+        Test unique constraint
+        a company and country combination can't be added more than once
+        """
+        company_1 = CompanyFactory()
+        canada = Country.objects.get(iso_alpha2_code='CA')
```

Comment:
Small thing but would suggest

```suggestion
        country = random_obj_for_model(Country)
```

as the particular country isn't important.